tests/framework/di/ContainerTest.php

```diff
@@ -286,4 +287,26 @@ public function testContainerSingletons()
         $sameFoo = $container->get('qux.using.closure');
         $this->assertSame($foo, $sameFoo);
     }
+
+    /**
+     * @requires PHP 5.6
```

Comment:
also https://github.com/sebastianbergmann/phpunit/issues/1984 this is equivalent to ask php version 5.6 or above